docs/generated/cli/exec.md

```diff
@@ -55,7 +63,7 @@ Ignore cycles in the task graph
 
 Type: `string`
 
-Choices: [dynamic, static, stream, stream-without-prefixes, compact]
+Choices: [dynamic, static, stream, stream-without-prefixes]
```

Comment:
Can this be removed from this? It doesn't get read.

Reply:
removed